src/styles/globals.css

```
@@ -0,0 +1,654 @@
+@import "tailwindcss";
+@plugin "@tailwindcss/typography";
+
+/* ═══════════════════════════════════════
+   THEME — font & screen tokens for Tailwind
+═══════════════════════════════════════ */
+@theme {
+  --font-sans: var(--font-epilogue), system-ui, sans-serif;
+  --font-serif: var(--font-fraunces), Georgia, serif;
```

Stylelint (17.3.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)